app/js/pages/WhatIsBlockstack.js

```diff
@@ -0,0 +1,247 @@
+import MultiVideoPlayer    from '../components/MultiVideoPlayer'
+import MailchimpForm       from '../components/MailchimpForm'
+import PostPreview         from '../components/PostPreview'
+import {featuredApps}      from '../config'
+import ModalVideo          from 'react-modal-video'
+
+function mapStateToProps(state) {
+  return {
+    posts: state.blog.posts,
+    stats: state.stats,
+  }
+}
+
+function mapDispatchToProps(dispatch) {
+  return bindActionCreators(
+    Object.assign({}, BlogActions, StatsActions),
+    dispatch
+  )
+}
+
+class HomePage extends Component {
+
+  constructor(props) {
+    super(props)
+
+    this.state = {
+      videoOpen: false,
+      stats: this.props.stats,
+      posts: this.props.posts
+    }
+
+    this.openModal = this.openModal.bind(this)
+  }
+
+  componentWillMount() {
+    if (this.props.posts.length === 0) {
+      this.props.fetchPosts()
+    }
+    this.props.fetchStats()
+  }
+
+  componentWillReceiveProps(nextProps) {
+    if (nextProps.stats !== this.props.stats) {
+      let stats = nextProps.stats
+      if (stats.domains === 0) {
+        stats.domains = 72000
+      }
+      this.setState({
+        stats: stats,
+      })
+    }
+
+    if (nextProps.posts !== this.props.posts) {
+      this.setState({
+        posts: nextProps.posts,
+      })
+    }
+  }
+
+  openModal() {
+    this.setState({ videoOpen: true })
+  }
+
+  render() {
+    const subscribeURL = '//blockstack.us14.list-manage.com/subscribe/post?u=394a2b5cfee9c4b0f7525b009&amp;id=0e5478ae86'
+
+    return (
+      <DocumentTitle title="What is Blockstack?">
+        <div>
+          <section className="jumbotron jumbotron-fluid text-center">
+            <div className="container">
+              <h1 className="page-heading mx-auto">
+                What is Blockstack?
+              </h1>
+              <p style={{ maxWidth: '410px' }} className="mb-5 mx-auto">
+                Blockstack helps people find and use apps that let them own their data. Developers can use Blockstack's platform to build these apps, also known as decentralized apps, to give their users a better experience.
+              </p>
+              <Link to="install" className="btn btn-primary">Download Blockstack</Link>
+            </div>
+          </section>
+
+          <section className="features">
+            <div className="container">
+              <div className="feature">
+                <div className="feature-number">1</div>
+                <div className="feature-body">
+                  <div className="row">
+                    <div className="col-md-6">
+                      <h2 className="font-weight-bold">Own your data</h2>
+                      <p>
+                        Right now, app users are "digital serfs" and applications are the "digital landlords". Users don't own their data; the app owns it. Users don't control where data gets stored; they’re only allowed to store it on the application.
```

Comment:
period inside quotes & change to smart quotes

the "digital landlords". -> the “digital landlords.“